### PR TITLE
MOTECH-1931: Fix ordering inconsistencies

### PR DESCRIPTION
**Pull request by pgesek**
* Allowed ordering on multiple fields using QueryParams
* In the data-browser, use ID ordering as backup to the provided
  one
* Cleaned up some duplications in InstanceController and MdsBundleIT

### CODE REVIEW

File: platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java [hunk 1]
```diff
@@ -331,4 +313,36 @@ private EntityRecord decodeBlobFiles(EntityRecord record) {
...
     }
+
+    private QueryParams buildQueryParams(GridSettings settings) {
+        return buildQueryParams(settings, buildOrderList(settings));
+    }
+
+    private QueryParams buildQueryParams(GridSettings settings, List<Order> orderList) {
+        // just check if the page is set
+        int page = (settings.getPage() == null) ? 1 : settings.getPage();
+        int pageSize = (settings.getPage() == null) ? 10 : settings.getRows();
```
Comment (sebbrudzinski) [inline platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java]: getRows(), I suppose

File: platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java [hunk 2]
```diff
@@ -331,4 +313,36 @@ private EntityRecord decodeBlobFiles(EntityRecord record) {
...
+
+        if (settings.getSortColumn() != null && !settings.getSortColumn().isEmpty()) {
+            order = new Order(settings.getSortColumn(), settings.getSortDirection());
+        }
+
+        List<Order> orderList = new ArrayList<>();
+        if (order != null) {
+            orderList.add(order);
+            if (!Constants.Util.ID_FIELD_NAME.equalsIgnoreCase(order.getField())) {
+                // if the ordering is done on a field other then id
```
Comment (sebbrudzinski) [inline platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java]: than

**Comment by sebbrudzinski** [comment]
Seems to work properly, I've left 2 small comments inline.

File: platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java (hunk 1 above)
Comment (pgesek) [inline platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/InstanceController.java]: I've left the logic for this as it was, but I guess we can use getRows here

**Pull request merged**